shell/packages/sandstorm-ui-powerbox/powerbox-client.js

```diff
@@ -504,11 +318,7 @@ Template.powerboxRequest.helpers({
 
   requestedInterfaceIsImplementedByFrontendRef: function () {
     const ref = Template.instance().data.get();
-    return (
-      ref.requestedInterfaceMatchesTag({ id: "12214421258504904768" }) || // IpNetwork
-      ref.requestedInterfaceMatchesTag({ id: "16369547182874744570" }) || // IpInterface
-      ref.requestedInterfaceMatchesTag({ id: "15831515641881813735" }) // UiView
-    );
+    return true;
```

Comment:
Oops, done. (I had modified this function quickly while debugging then forgot about it...)